No comment injection for latex grammar

### Summary

Currently, the injection of comment highlights which add highlights such as `#1234` for issue numbers `@username` for usernames and perhaps most importantly `TODO` and friends for comments related to bugs and todos are not correctly injected for LaTeX.

It seems that this was partially fixed in #4528, however I noticed that the `highlights.scm` query for LaTeX also mentions `comment_environment` and `block_comment`, and if we highlight those as comments then we also want comment injections for them.

### Reproduction Steps

<!-- Ideally provide a key sequence and/or asciinema.org recording. --> 

I tried this:
1. Create a file like this:
```latex
\begin{document}
	Lorem Ipsum
\end{document}


\begin{comment}
	comment invironment
	FIXME: lorem @username ipsum #12333
\end{comment}


\iffalse
	block comment
	FIXME: lorem @username ipsum #12333
\fi


% line comment
% FIXME: lorem @username ipsum #12333

```
3. `hx <file>`

I expected this to happen:
Things such as `TODO` or `@username` are highlighted as for languages like rust or bash for all comment types.

Instead, this happened:
Things such as `TODO` or `@username` are only highlighted for `line_comment`s on master and not at all on stable.

### Helix log

<details><summary>~/.cache/helix/helix.log</summary>

```
2022-11-28T11:50:14.937 helix_view::theme [WARN] Theme: malformed hexcode: fg
2022-11-28T11:50:14.941 helix_view::editor [ERROR] Failed to initialize the LSP for `source.tsq` { LSP not defined }
2022-11-28T11:51:06.674 helix_view::theme [WARN] Theme: malformed hexcode: fg
2022-11-28T11:51:06.680 helix_view::editor [ERROR] Failed to initialize the LSP for `source.tsq` { LSP not defined }
2022-11-28T11:51:58.949 helix_view::theme [WARN] Theme: malformed hexcode: fg
2022-11-28T11:51:58.954 helix_view::editor [ERROR] Failed to initialize the LSP for `source.tsq` { LSP not defined }
2022-11-28T11:53:36.276 helix_view::theme [WARN] Theme: malformed hexcode: fg
2022-11-28T11:53:36.283 helix_view::editor [ERROR] Failed to initialize the LSP for `source.tsq` { LSP not defined }
2022-11-28T11:59:43.518 helix_view::theme [WARN] Theme: malformed hexcode: fg
2022-11-28T12:00:17.727 helix_view::theme [WARN] Theme: malformed hexcode: fg
```

</details>


### Platform

Linux 6.0.9-arch1-1

### Terminal Emulator

alacritty 0.11.0

### Helix Version

helix 22.08.1

--- a/runtime/queries/latex/injections.scm
+++ b/runtime/queries/latex/injections.scm
@@ -1,2 +1,6 @@
-((line_comment) @injection.content
- (#set! injection.language "comment"))
+([
+   (comment)
+   (line_comment)
+   (block_comment)
+   (comment_environment)
+ ] @injection.content (#set! injection.language "comment"))

